packages/contractkit/src/kit.test.ts

```diff
@@ -0,0 +1,101 @@
+import { mockContractAddress } from '@celo/test-utils'
+import Web3 from 'web3'
+import { CeloContract, ContractKit, newKit, newKitFromWeb3 } from '.'
+import { AddressRegistry } from './address-registry'
+import { WrapperCache } from './contract-cache'
+import { Web3ContractCache } from './web3-contract-cache'
+
+describe('ContractKit', () => {
```

Comment:
a nice thing about jest is that you can remove the parent `describe`. totally optional though

Reply:
wont that be less... descriptive?